src/nanoemoji/colors.py

```diff
@@ -185,20 +186,54 @@ def color_name(rgb) -> Optional[str]:
     return None
 
 
-class Color(collections.namedtuple("Color", "red green blue alpha")):
-    _CURRENT_COLOR = (-1, -1, -1)
+@dataclasses.dataclass(frozen=True, order=True)
+class Color(Sequence):
+    red: int
+    green: int
+    blue: int
+    alpha: float = 1.0
+    index: Optional[int] = None
+
+    _CURRENT_COLOR: ClassVar[Tuple[int, int, int]] = (-1, -1, -1)
```

Comment:
Shouldn't this be an instance of Color?

Reply:
Color class hasn't completely been defined yet, can't make an instance of it from within the class scope (btw this code preexists my PR)

Reply:
There must be a way to make it an actual instance? Make a method akin to Affine2D.identity()? (nice for consistency too)